source/support.tex

```diff
@@ -2203,9 +2203,9 @@
 \begin{note}
 A function registered via \tcode{at_quick_exit}
 is invoked by the thread that calls \tcode{quick_exit},
-which can be a different thread
-than the one that registered it, so registered functions cannot rely on the identity
-of objects with thread storage duration.
+which can be a different thread than the one that registered it,
```

Comment:
"can"?

Reply:
How about "is invoked in the thread that calls `at_quick_exit`, a possibly different one than the one that registered it"?

Reply:
calls quick_exit not at_quick_exit, and maybe "a possibly different thread than the one" to avoid "one than the one".

Reply:
I don't think we need a change here, I'm happy to resolve this comment